src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/modifier/redundantmodifier/Example1.java

```diff
@@ -0,0 +1,43 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="RedundantModifier"/>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.modifier.redundantmodifier;
+
+// xdoc section -- start
+public class Example1 {
+
+  void test() {
+    // violation below, 'Redundant 'final' modifier'
+    try (final var a = lock()) {
+
+    } catch (Exception e) {
+
+    }
+  }
+
+  // violation below, 'Redundant 'abstract' modifier'
+  abstract interface I {
+    public abstract void m();
+    // 2 violations above:
+    //                   'Redundant 'public' modifier'
+    //                   'Redundant 'abstract' modifier'
```

Comment:
Please make indentation to be 4 space to make code less wide and fit phone screen:)